packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -50,3 +76,132 @@ void writeFlutterPluginsList() {
     }
   }
 }
+
+const String _androidPluginRegistryTemplate = '''package io.flutter.plugins;
+
+import io.flutter.app.FlutterActivity;
+
+{{#plugins}}
+import {{package}}.{{class}};
+{{/plugins}}
+
+/**
+ * This file is generated. Do not edit.
+ */
+
+public class PluginRegistry {
+{{#plugins}}
+    {{class}} {{name}};
```

Comment:
I think maybe the visibility of these should be explicitly specified. If you want them to be public then consider making them private and providing a getter so it's clear that they're not meant to be mutated externally. Or maybe have them be final and set in the PluginRegistry's constructor.